### PR TITLE
Removing header guard under scores for all files under RIOT/boards

### PR DESCRIPTION
**Pull request by BigDaddyD**
This PR is to in part repair the outdated header include guards. This PR is for all cases under the RIOT/boards directory and all consequent subdirectories.

### CODE REVIEW

**Comment by authmillenon** [comment]
Please rename the PR to something less referential (e. g. "all: fix header guards" or "all: remove heading underscores for header guards")

**Comment by authmillenon** [comment]
Since this also touches some board/cpu internal defines I reassign to someone with more experience of MSP-430 architecture. One word though: I encountered the following guard format in `sys/` mostly: `FILENAME_H_` (note the trailing underscore). For consistency it would make sense to use this format here too.

File: boards/mbed_lpc1768/system.c [hunk 1]
```diff
@@ -382,27 +382,27 @@
...
 
 /* F_cco0 = (2 * M * F_in) / N  */
-#define __M               (((PLL0CFG_Val      ) & 0x7FFF) + 1)
-#define __N               (((PLL0CFG_Val >> 16) & 0x00FF) + 1)
-#define __FCCO(__F_IN)    ((2ULL * __M * __F_IN) / __N)
-#define __CCLK_DIV        (((CCLKCFG_Val      ) & 0x00FF) + 1)
+#define M               (((PLL0CFG_Val      ) & 0x7FFF) + 1)
+#define N               (((PLL0CFG_Val >> 16) & 0x00FF) + 1)
+#define FCCO(__F_IN)    ((2ULL * __M * __F_IN) / __N)
+#define CCLK_DIV        (((CCLKCFG_Val      ) & 0x00FF) + 1)
```
Comment (authmillenon) [inline boards/mbed_lpc1768/system.c]: `mbed_lpc1768` builds fail due to this changes.

**Comment by BigDaddyD** [comment]
Is that trailing underscore for all guards within board/ ?

**Comment by authmillenon** [comment]
I'd say so.

**Comment by BigDaddyD** [comment]
Should we update the issue as a whole?

**Comment by BigDaddyD** [comment]
Since this PR has errors and we want to move from no underscore to a trailing underscore, this request should probably be closed and I can create a new one with the new style

**Comment by authmillenon** [comment]
I don't think that this is necessary, just adapt the commits accordingly.

**Comment by OlegHahm** [comment]
Are you planning to do so, @BigDaddyD?

**Comment by OlegHahm** [comment]
Superseded by #3056

**Pull request closed without merging**